app/src/androidTest/java/ch/epfl/sdp/drone3d/ItineraryCreateActivityTest.kt

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
I think that there were already a few tests for this class but with the wrong name (MapActivityTest). It would be good to regroup all of them into this class.